ow_faults_detection/src/FaultDetector.cpp

```diff
@@ -8,10 +8,130 @@
 using namespace std;
 using namespace ow_lander;
 
+constexpr std::bitset<10> FaultDetector::isCamExecutionError;
+constexpr std::bitset<10> FaultDetector::isPanTiltExecutionError;
+constexpr std::bitset<10> FaultDetector::isArmExecutionError;
+constexpr std::bitset<10> FaultDetector::isPowerSystemFault;
+
+constexpr std::bitset<3> FaultDetector::islowVoltageError;
+constexpr std::bitset<3> FaultDetector::isCapLossError;
+constexpr std::bitset<3> FaultDetector::isThermalError;
+
 FaultDetector::FaultDetector(ros::NodeHandle& node_handle)
 {
-  
   srand (static_cast <unsigned> (time(0)));
-  m_pub = node_handle.advertise<ow_faults::SystemFaults>("/test", 10);
+
+ auto image_trigger_str = "/StereoCamera/left/image_trigger";
+  m_camera_original_trigger_sub = node_handle.subscribe(string("/_original") + image_trigger_str,
+    10, &FaultDetector::cameraTriggerOriginalCb, this);
```

Comment:
I didn't expect the FaultDetector node to make any references to the **_original** topic(s).